RecoCTPPS/ProtonReconstruction/python/ctppsProtons_cff.py

```diff
@@ -1,7 +1,47 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoCTPPS.ProtonReconstruction.ctppsProtons_cfi import *
+# import default alignment settings
+from CalibPPS.ESProducers.ctppsAlignment_cff import *
 
-# TODO: remove these lines once conditions data are available in DB
+# import default optics settings
 from CalibPPS.ESProducers.ctppsOpticalFunctions_cff import *
+
+# import and adjust proton-reconstructions settings
+from RecoCTPPS.ProtonReconstruction.ctppsProtons_cfi import *
 ctppsProtons.lhcInfoLabel = ctppsLHCInfoLabel
+
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+
+def ApplyDefaultSettings(ctppsProtons):
```

Comment:
Start with a lowercase letter, as for cms code rules: ```applyDefaultSettings``` (and the same for the similar cases below)

Reply:
Fixed in d8eb24c.